ui/src/components/InlineHelp/__tests__/InlineHelp.test.ts

```diff
@@ -0,0 +1,21 @@
+import { shallowMount } from "@vue/test-utils";
+import InlineHelp from '../InlineHelp.vue';
+import { assert } from "chai";
+
+describe("InlineHelp test scripts", () => {
+    test("Renders correctly", () => {
```

Comment:
Just for consistency: We have always used `it(…)` so far instead of `test(…)`. I think they are the same. Do you mind changing that?